src/components/ListingPage/styles.ts

```diff
@@ -32,54 +20,77 @@ export const FiltersContainer = styled.div`
 `;
 
 export const PageContainer = styled.div`
-  height: 100vh;
+  height: 100svh;
   width: 100%;
   display: flex;
   flex-direction: column;
-  padding-top: 1rem;
+
+  /* customize scrollbar for listing pages only */
```

Comment:
@kyrenetam should we add a custom scrollbar to other pages?

Reply:
from final worksesh:
[alvaro] that looks nasty